src/sass/_basics.scss

```diff
@@ -1,12 +1,15 @@
 $include-html: false !default;
 
 @if ($include-html) {
+  html {
```

Comment:
Gives the ability to change this value and get **all** components scaled evenly.

Together with `$baseline` defines vertical grid basis vector magnitude.